cla_backend/apps/legalaid/models.py

```diff
@@ -158,6 +158,8 @@ class PersonalDetails(CloneModelMixin, TimeStampedModel):
     # only normalised version of post code for now
     search_field = models.TextField(null=True, blank=True, db_index=True)
 
+    announce_call = models.BooleanField(blank=False, default=False)
```

Comment:
@BenMillar-CGI Should this be a null boolean field? So that old records can have a default value of null

Reply:
Hi @said-moj, this implementation defaults all existing records to False, which was the requested behaviour. However, the frontend will display the warning whether the value is null or False. 

I am happy to update this if you think this is the best way to approach it 👍

Reply:
Hi @said-moj I have updated this model to use `NullBooleanField` as suggested. Many thanks!! 😄